credentialAtomicQueryMTP.go

```diff
@@ -1,208 +1,208 @@
-)
-
-// LevelsAtomicQueryMTPCircuit is number of merkle tree levels credentialAtomicQuery.circom compiled with
-const LevelsAtomicQueryMTPCircuit = 40
-
-// ValueArraySizeAtomicQueryMTPCircuit size of value array
-const ValueArraySizeAtomicQueryMTPCircuit = 16
-
-// AtomicQueryMTP represents credentialAtomicQuery.circom
-type AtomicQueryMTP struct{}
-
-// AtomicQueryMTPInputs represents input data for kyc and kycBySignatures circuits
+// AtomicQueryMTPInputs ZK private inputs for credentialAtomicQueryMTP.circom
 type AtomicQueryMTPInputs struct {
+	BaseConfig
 	// auth
-	ID                 *core.ID
-	AuthClaim          Claim
-	AuthClaimRevStatus RevocationStatus
-	Challenge          *big.Int
-	Signature          *babyjub.Signature
+	ID        *core.ID
+	AuthClaim Claim
+	Challenge *big.Int
+	Signature *babyjub.Signature
 
-	CurrentStateTree TreeState
+	Claim // claim issued for user
 
-	// issuerClaim
-	Claim
-	RevocationStatus
+	Schema           core.SchemaHash
```

Comment:
we agreed to move Schema to Claim tyoe, didn't we?

Reply:
yes, I missed this